src/App.js

```diff
@@ -2,15 +2,49 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    number: 0,
+  };
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState(prevState => ({
+      number: prevState.number + 100,
+    }));
+  };
 
-  increase = () => {};
+  addOne = () => {
+    this.setState(prevState => ({
+      number: prevState.number + 1,
+    }));
+  };
+
+  increase = () => {
+    this.addOne();
+    if (this.state.number % 5 === 0) {
```

Comment:
```suggestion
    if (this.state.number % 5 === 0 && this.state.number !== 0) {
```
Можно добавить еще такую проверку